packages/tables/resources/lang/fr/table.php

```diff
@@ -16,6 +16,14 @@
 
     'fields' => [
 
+        'bulk_select_page' => [
+            'label' => 'Sélectionner/déselectionner tous les éléments pour les actions groupées.',
+        ],
+
+        'bulk_select_record' => [
+            'label' => 'Sélectionner/désélectionner l\'élément :key pour les actions groupées.',
```

Comment:
Could you use `"` instead of `'` here to prevent having to escape in the string?

Reply:
Sure! Some other strings used `\` so I went with that.